src/js/__tests__/__snapshots__/metadata-test.js.snap

```diff
@@ -1898,8 +1896,8 @@ Object {
     "reactjs",
   ],
   "Keyboard": Array [
-    "accessibility",
-    "tab",
+    "keyboard",
```

Comment:
Why was 'accessibility' and 'tab' got removed? we should keep them in addition to the new keywords.